histoqc/SaveModule.py

```diff
@@ -19,6 +20,96 @@ def blend2Images(img, mask):
     return out
 
 
+'''
+the followings are helper functions for generating geojson
+'''
+
+feature_template = {
```

Comment:
Yes, you are right. I will use python-geojson and cv2geojson. I will let you know if there is any issue with using them.

Reply:
Great! note though that we want to avoid adding dependencies unless necessary.

I would argue that the `geojson` library is the right tool for the job in this case, but I'm not sure yet with `cv2geojson` - do you think this second library has much added value for HistoQC?

Also, do you have a sense of how large these packages are?